pkg/odo/cli/component/create.go

```diff
@@ -910,12 +910,16 @@ func (co *CreateOptions) downloadProject(projectPassed string) error {
 		return errors.Errorf("Project type not supported")
 	}
 
+	log.Info("Project")
```

Comment:
You need to add `\n` within this. See output, as there is no spacing between Validation and Project

```sh
~/openshift/f3                                                                                                                                                                                                                                                                   
▶ odo create --starter nodejs
Experimental mode is enabled, use at your own risk

Validation
 ✓  Checking devfile existence [155676ns]
 ✓  Checking devfile compatibility [452063ns]
 ✓  Creating a devfile component from registry: DefaultDevfileRegistry [323241ns]
 ✓  Validating devfile component [357302ns]
Project
 ✓  Downloading project from https://codeload.github.com/odo-devfiles/nodejs-ex/legacy.zip/master [294ms]

Please use `odo push` command to create the component with source deployed
```

Reply:
updated